src/main/jbake/content/blog/2017/03/15/hawkular-opentracing-otsc.adoc

```diff
@@ -0,0 +1,44 @@
+= Distributed Tracing Workshop and OpenTracing Collaboration
+Pavol Loffay
+2017-03-15
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+The Distributed tracing workshop is organized by the
+https://groups.google.com/forum/#!forum/distributed-tracing[Distributed Tracing Workgroup]
+and takes place approximately once a year. This time it was 18th-19th February in Berlin. It was a pleasure
+to be there with people from various companies and share ideas and a common goal to improve distributed tracing.
+
+== Summary
+
+We discussed many things, and here are my main takeaways:
```

Comment:
This line ends with a colon, but then so do the top level points below, so wondering whether better if the two points below should also be bullet points, with the subpoints indented a level?